docs/CONTRIBUTE.md

```diff
@@ -316,7 +316,7 @@ fine.
 This means that all files need to have their license and copyright information
 clearly stated. Ideally by having the standard curl source code header, with
 the SPDX-License-Identifier included. If the header does not work, you can use a
-smaller header or add the information for a specific file to the `.reuse/dep5`
+smaller header or add the information for a specific file to the `REUSE.toml`
```

Comment:
I suppose there is no way to *not* add a new file to the root directory? The previous version was cleverly "hiding" there due to the leading dot in the name.

Reply:
Unfortunately that is not possible by design. A REUSE.toml can only describe files that are next to it or in child directories.

Reply:
I was afraid of that. It is a common "decease": adding config files to the source root. It does not scale and is user hostile. But I realize we need to live with this now.

I propose that we then add a comment at the top of this file that explains its purpose and why it is there so that users finding the file and checking it out have a chance to be less confused about it.

Reply:
Understood. I have amended the REUSE.toml file accordingly.

The REUSE project team is aware of this downside but all other options were worse after all, especially when aiming to make the specification still maintainable and logical for humans and tools. It follows the info-as-close-to-the-file-as-possible principle.

Reply:
It would be nice if we could name it `.reuse.toml` (or `.REUSE.toml`).